app/services/list_tasks.rb

```diff
@@ -0,0 +1,24 @@
+class ListTasks < ListCollection
+
+  attr_defaultable :task_respository, -> { Task }
+  attr_defaultable :result_serializer, -> { V1::TaskSerializer }
+
+
+  attr_reader :project_id
+
+
+  def initialize project_id: nil, page: 1, page_size: 25
+    super()
```

Comment:
If you leave the `()` off then all arguments will be passed to the super class initializer. If you explicitly say `()` then the superclass won't get any arguments. Neither of those is likely what you want and instead you likely want to pass the `page` and `page_size` up to the super class: `super(page: page, page_size: page_size)`